### DISCUSSION TITLE
Too many warnings when saving the submarine in the editor can push the "Ok" button off-screen

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#13847] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

If the player somehow makes too many warnings appear when they save the submarine in the sub editor, the list of warnings will push the OK button off-screen. The message can still be closed by pressing Enter.
It'd be better if the size of the window was capped and the text inside was scrollable.

![602960_20240426161115_1](https://github.com/FakeFishGames/Barotrauma/assets/47554589/d3f5d510-4f12-4866-8a32-e4201e730187)

### Reproduction steps

1. Open a submarine file in the submarine editor.
2. Create problems (vents with not enough oxygen, lack of certain items and tags).
3. Save the submarine to see the list of warnings.
4. Repeat until the list becomes too long.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.4.4.1 (Blood in the Water Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I [created a ticket about this in the issue tracker](https://github.com/FakeFishGames/Barotrauma/issues/13850), we'll look into it!

**Comment by Rokvach** [comment]
Issue has been fixed and tested, now all the warnings are contained inside a scrollable list. Working correctly, closing discussion.
